ftest.sh

```diff
@@ -267,7 +267,8 @@ else
     cd $DAOS_BASE
 fi
 
-export CRT_PHY_ADDR_STR=ofi+sockets
+# export CRT_PHY_ADDR_STR=ofi+sockets
+export CRT_PHY_ADDR_STR="ofi+verbs;ofi_rxm"
```

Comment:
(lint) This is actually an end quote, but due to next char it looks suspect. [SC1079]